api/api_ethereum.go

```diff
@@ -983,8 +983,8 @@ func newEthTransactionReceipt(tx *types.Transaction, b Backend, blockHash common
 
 	if b.ChainConfig().IsEthTxTypeForkEnabled(new(big.Int).SetUint64(blockNumber)) {
 		// TODO-Klaytn: Klaytn is using fixed BaseFee(0) as now but
-		// if we apply dynamic BaseFee, we should add calculated BaseFee instead of using params.BaseFee.
-		baseFee := new(big.Int).SetUint64(params.BaseFee)
+		// if we apply dynamic BaseFee, we should add calculated BaseFee instead of using params.ZeroBaseFee.
+		baseFee := new(big.Int).SetUint64(params.ZeroBaseFee)
```

Comment:
Here also we need to return actual value.

Reply:
I will resolve those for now, because rpc api will be implemented 2nd development.